core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -733,6 +733,15 @@ private static int getSlaveAgentPortInitialValue(int def) {
      */
     private transient final AdjunctManager adjuncts;
 
+    /**
+     * Interval in seconds between 2 ajax calls.
+     */
+    private int ajaxRefreshInterval = 6;
+    public int getAjaxRefreshInterval() { return this.ajaxRefreshInterval;}
+    public void setAjaxRefreshInterval(int interval) {
```

Comment:
YAGNI :)

Reply:
I was trying to add a UI configuration part in `/configure` but I really don't understand how it works, if I move to system property I can remove it I agree.

Reply:
generally tunables are not in configuration as we do not generally expect them to be tuned.
https://www.jenkins.io/doc/book/managing/system-properties/  (would also need a PR for the new property)